wordle-web/pages/game.vue

```diff
@@ -12,15 +12,17 @@
     <v-card-text class="text-h1 font-weight-black text-center">
       Wordle!
     </v-card-text>
-
+    
     <v-alert v-if="wordleGame.gameOver" width="80%" :type="gameResult.type">
       {{ gameResult.text }}
       <v-btn class="ml-2" @click="resetGame"> Play Again? </v-btn>
+      <v-btn class="ml-2" @click="savePlayerScore"> Save Game </v-btn>
     </v-alert>
 
     <game-board :wordleGame="wordleGame" />
 
     <keyboard :wordleGame="wordleGame" />
+    <createUser :wordleGame="wordleGame" :dialog="showSaveScoreDialog" />
```

Comment:
```suggestion
    <createUser :wordleGame="wordleGame" :dialog.sync="showSaveScoreDialog" />
```

When the property you are passing to a child component is going to be updated by the child, you want to make sure you specify that it should be kept in sync with the parent component.